www/class/centreonContactgroup.class.php

```diff
@@ -201,6 +201,85 @@ public function insertLdapGroup($cg_name)
         $this->db->query($queryCacheLdap);
         return $row['cg_id'];
     }
+    
+    /**
+     * Synchronize with LDAP groups at config generation
+     *
+     * @return array | array of error messages
+     */
+    public function syncWithLdapConfigGen()
+    {
```

Comment:
For this function, I find the commentary too light.
The code is hard to read, to understand, you have to read line by line.
The name of the variables is not very explicit. 
Would it be possible to divide certain parts by block or function? Or at least comment on several places in the code?

Reply:
I forked a PR deployed on the customer's prod.
But this part of the code is from : 
https://github.com/centreon/centreon/pull/7130/files
As it's still waiting for QA validation, I couldn't do an other way without creating multiple conflicts. And this part of the PR needed to be fixed too.